alembic/versions/614911daf883_add_seasonal_to_time_resolution_enum.py

```diff
@@ -0,0 +1,145 @@
+"""add 'seasonal' to time_resolution enum
+
+Revision ID: 614911daf883
+Revises: 
+Create Date: 2017-07-17 17:00:43.066818
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '614911daf883'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+connection = op.get_bind()
+dialect = connection.dialect.name
+
+
+# This is a *highly* customized migration script, barely related to the
+# autogenerated script at all. Two reasons:
+#
+# 1) There are a lot of differences between the (test, and presumably prod)
```

Comment:
This turns out to be an important question. 

- When I used Alembic to autogenerate a migration from the SQLite test database (`modelmeta/data/mddb-v2.sqlite`), it detected a lot of differences (see code snippet below), most of which were undeclared contents of the database (a table, several indexes).
- When I did the same thing with a locally hosted copy of the production database made with the script `scripts/copyproddb.py`, it detected a similar but more extensive set of differences (see code snippet below). 
  - That database isn't necessarily a _perfect_ copy of the prod database (the script is a little complex), but it has got to be pretty close. 
- I haven't done the same test with the production database because Alembic always creates its `alembic_version` table if it doesn't exist, and (a) I don't think have write permission for the prod database, and (b) I'm leery of modifying the prod database without consulting you first.

Autogenerated upgrade from current SQLite test db to current SQLAlchemy declaration:
```
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('spatial_ref_sys')
    with op.batch_alter_table('climatological_times', schema=None) as batch_op:
        batch_op.drop_index('climatological_times_time_set_id_key')

    with op.batch_alter_table('data_files', schema=None) as batch_op:
        batch_op.drop_index('data_files_run_id_key')
        batch_op.drop_index('data_files_unique_id_key')

    with op.batch_alter_table('ensembles', schema=None) as batch_op:
        batch_op.drop_index('ensemble_name_version_key')

    with op.batch_alter_table('grids', schema=None) as batch_op:
        batch_op.drop_constraint('grid_srid_fkey', type_='foreignkey')
        batch_op.drop_column('srid')

    with op.batch_alter_table('runs', schema=None) as batch_op:
        batch_op.drop_index('runs_emission_id_key')
        batch_op.drop_index('runs_model_id_key')
        batch_op.drop_index('unique_run_model_emissions_constraint')

    with op.batch_alter_table('times', schema=None) as batch_op:
        batch_op.drop_index('time_set_id_key')

    with op.batch_alter_table('y_cell_bounds', schema=None) as batch_op:
        batch_op.drop_index('y_c_b_grid_id_key')

    # ### end Alembic commands ###
```

Autogenerated upgrade from current copy of production db to current SQLAlchemy declaration:
```
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('spatial_ref_sys')
    with op.batch_alter_table('climatological_times', schema=None) as batch_op:
        batch_op.drop_index('climatological_times_time_set_id_key')

    with op.batch_alter_table('data_file_variables', schema=None) as batch_op:
        batch_op.drop_constraint('data_file_variables_data_file_id_fkey', type_='foreignkey')
        batch_op.create_foreign_key(None, 'data_files', ['data_file_id'], ['data_file_id'])

    with op.batch_alter_table('data_file_variables_qc_flags', schema=None) as batch_op:
        batch_op.drop_constraint('data_file_variables_qc_flags_data_file_variable_id_fkey', type_='foreignkey')
        batch_op.create_foreign_key(None, 'data_file_variables', ['data_file_variable_id'], ['data_file_variable_id'])

    with op.batch_alter_table('data_files', schema=None) as batch_op:
        batch_op.drop_index('data_files_run_id_key')
        batch_op.drop_constraint('data_files_unique_id_key', type_='unique')

    with op.batch_alter_table('ensemble_data_file_variables', schema=None) as batch_op:
        batch_op.drop_constraint('ensemble_data_file_variables_ensemble_id_fkey', type_='foreignkey')
        batch_op.drop_constraint('ensemble_data_file_variables_data_file_variable_id_fkey', type_='foreignkey')
        batch_op.create_foreign_key(None, 'ensembles', ['ensemble_id'], ['ensemble_id'])
        batch_op.create_foreign_key(None, 'data_file_variables', ['data_file_variable_id'], ['data_file_variable_id'])

    with op.batch_alter_table('ensembles', schema=None) as batch_op:
        batch_op.drop_constraint('ensemble_name_version_key', type_='unique')

    with op.batch_alter_table('grids', schema=None) as batch_op:
        batch_op.drop_constraint('grid_srid_fkey', type_='foreignkey')
        batch_op.drop_column('srid')

    with op.batch_alter_table('runs', schema=None) as batch_op:
        batch_op.drop_index('runs_emission_id_key')
        batch_op.drop_index('runs_model_id_key')
        batch_op.drop_constraint('unique_run_model_emissions_constraint', type_='unique')

    with op.batch_alter_table('times', schema=None) as batch_op:
        batch_op.drop_index('time_set_id_key')

    with op.batch_alter_table('y_cell_bounds', schema=None) as batch_op:
        batch_op.drop_index('y_c_b_grid_id_key')
        batch_op.drop_constraint('y_cell_bounds_grid_id_fkey', type_='foreignkey')
        batch_op.create_foreign_key(None, 'grids', ['grid_id'], ['grid_id'])

    # ### end Alembic commands ###
```